### PR TITLE
Add steps to get CRN sites & download some example data

### PR DESCRIPTION
**Pull request by lindsayplatt**
Updated the pipeline to include steps to pull down [Climate Response Network](https://groundwaterwatch.usgs.gov/CRNHome.asp) sites and corresponding data. To use this pipeline and get data to play with, change the dates in `0_config.yml` to the time period you would like and run the following:

```r
library(scipiper)
scmake("1_fetch/out/gw_crn_data.rds")
crn_data <- readRDS("1_fetch/out/gw_crn_data.rds")
```

This adds CRN sites to fetch, but nothing else. The remaining process, visualize, and animate steps are the same.

### CODE REVIEW

**Comment by collnell** [comment]
hmmm I am getting this issue 
image

**Comment by lindsayplatt** [comment]
Can you verify that `gw_crn_sites` is not empty? `gw_crn_sites <- scmake("gw_crn_sites")`

**Comment by collnell** [comment]
It is empty

**Comment by lindsayplatt** [comment]
is `gw_crn_sites_sf` also empty?

**Comment by collnell** [comment]
No

**Comment by collnell** [comment]
But gw_crn_sites is null

**Comment by collnell** [comment]
Works with changing 
``` 
extract_crn_sites <- function(gwcrn_sf) {
  # The site ids are read in as factors in st_read
  gwcrn_sf %>% pull(SITEID) %>% levels()
}
```

to 
```
extract_crn_sites <- function(gwcrn_sf) {
  # The site ids are read in as factors in st_read
  gwcrn_sf %>% pull(SITEID) %>% unique()
}
```

File: 1_fetch/src/fetch_gwcrn.R [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  # Unzip the shapefile zip
+  fns <- unzip(zipfile = zipfile, overwrite = TRUE, exdir = tmp_dir)
+  
+  # Read and return an sf object of the shapefile
+  st_read(fns[grep(".shp", fns)])
+}
+
+extract_crn_sites <- function(gwcrn_sf) {
+  # The site ids are read in as factors in st_read
+  gwcrn_sf %>% pull(SITEID) %>% levels()
```
Comment (collnell) [inline 1_fetch/src/fetch_gwcrn.R:11]: change `levels()` to `unique()`

**Review by collnell — CHANGES_REQUESTED** [review]
It's working for me with `unique()` instead of `levels()`

**Comment by lindsayplatt** [comment]
Huh, wonder if there is a change in from R versions (I am using a pretty old version)

**Review by collnell — APPROVED** [review]

**Comment by collnell** [comment]
for me `levels()` only works with factors and not character

**Comment by lindsayplatt** [comment]
Huh, maybe that is the issue then. That column is a factor for me. I might have an old sf?

**Comment by collnell** [comment]
I think the default in R 4.0 is that strings are strings and no more stringsAsFactors

**Comment by lindsayplatt** [comment]
... I am still on 3.6 😳 guess I should update soon!

**Pull request merged**